### ISSUE TITLE
Minetest shows black screen running in the Enlightenment (E19) window manager.

### ISSUE THREAD

**Issue by JadedCtrl** [minetest/minetest#2965]
Upon starting Minetest, the game window only shows a black screen, not the main menu.
When it is started from the terminal, there are sound errors.
When I launch it from Xfce4, rather than Enlightenment, Minetest works just fine.
It seems that a few other people have had this problem.
I'm running GNU GuixSD v0.8.3.

Links to other examples of this issue:
https://forum.minetest.net/viewtopic.php?f=6&t=9809
http://irc.minetest.ru/minetest/2015-04-17#i_4226322     [At 21:57 by stormchaser3000]
http://irc.minetest.ru/minetest/2014-07-23#i_3827691      [At 15:19 by Will_McDade]

Screenshots:
![shot-2015-07-27_20-56-00](https://cloud.githubusercontent.com/assets/10477760/8917632/c0a1c70e-34a2-11e5-959f-d7ea098ad0b9.jpg)
![2015-07-27-210059_1600x900_scrot](https://cloud.githubusercontent.com/assets/10477760/8917644/d953fd4e-34a2-11e5-8045-548e0d6823a0.png)

**Comment by asl97** [comment]
1. you can directly link to the message by clicking the time in the irc chat log

http://irc.minetest.ru/minetest/2015-04-17#i_4226322
http://irc.minetest.ru/minetest/2014-07-23#i_3827691

2. from the screenshot, i am going to assume that you got it from the package manager,  i would suggest building a debug build yourself 

3. the sound error is most likely cause it doesn't have pulse, alsa or oss,  what does it use for sound?  jackd?

**Comment by JadedCtrl** [comment]
1- Thanks, asl. I added the direct links.
3- In compiling Minetest, I found out that I didn't openal libvorbis installed. Sound works now.
2- And yea, I got it from the package manager. I've compiled the latest Minetest from GitHub, and got the same results.

4- I found the reason Minetest wasn't rendering-- it seems to be incompatible with some of the composite features of Enlightenment, or vice versa. The way to have Minetest run on Enlightenment is to either disable composite effects, switch to software rendering, or disable tear-free rendering.

Thanks, asl. :)
I'm not sure if this is a bug in Minetest, or a bug in Enlightenment. I guess I'll go ahead and close it?
